### PR TITLE
Added Payments Setting tab in the settings 

### PR DESCRIPTION
**Pull request by aniket-k-kaushik**
## Notion card
- [x] New feature (non-breaking change which adds functionality)
### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [x] I have added automated tests for my code

### CODE REVIEW

File: app/views/payments_setting/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<H1>Code</H1>
```
Comment (keshavbiswa) [inline app/views/payments_setting/index.html.erb:1]: I'm not sure what this is supposed to be?

File: app/controllers/payments_setting_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class PaymentsSettingController < ApplicationController
```
Comment (keshavbiswa) [inline app/controllers/payments_setting_controller.rb:3]: Shouldn't this be PaymentSettings controller?

File: config/routes.rb [hunk 3]
```diff
@@ -62,6 +62,9 @@ def draw(routes_name)
   get "payments/*path", to: "payments#index", via: :all
   get "payments", to: "payments#index"
 
+  get "payments_setting/*path", to: "payments_setting#index", via: :all
+  get "payments_setting", to: "payments_setting#index"
```
Comment (keshavbiswa) [inline config/routes.rb:66]: I'm not understanding why are we doing this and not 
```ruby
resource :payment_settings, only: [:index]
```

Is there a specific use case I'm missing?

**Review by keshavbiswa — COMMENTED** [review]
Added some comments.

File: app/controllers/payments_setting_controller.rb (hunk 2 above)
Comment (aniket-k-kaushik) [reply app/controllers/payments_setting_controller.rb:3]: I can change it to paymentSettings,

**Pull request merged**